smoke-test-app/config/ember-try.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  useYarn: true,
```

Comment:
why do we declare yarn here but then refine the lockfile and replace it with package.lock?

Reply:
So the changes to the lockfile were to remove the hacky use of the packed npm package that was actually defined in the lock file. Having it there causes a lot of problems for npm. You can see by my work here https://github.com/ember-cli/ember-try/pull/281/files#diff-6d3184bf759343ad27b80888330c4ba4R10 that there needs to be slightly different behaviour between how npm does this and yarn does this.

The reason why I'm adding a package-lock.json is because the bug that this PR fixes only happens if there is a package-lock in the project that ember-try is working on. To be able to test in yarn and npm effectively we need to have lock files for both environments.

I added the `useYarn` to the configs because before this PR they were doing ember-try with yarn and while they **did** work with npm I didn't want to change the behaviour of the CI suite just to add support for npm 👍

Reply:
The smoke tests were using `npm` before though -- if you don't have `useYarn:true`, it uses `npm`

Reply:
I don't know why I thought the tests were using `yarn` before 🙈 I have removed this change